Download of small volume tracings produces 0 b data.zip

Through chance, I noticed that duplicating said small tracing resulted in an exception on the tracingstore:

```
 ! @7a6d5k2a8 - Internal server error, for (GET) [/tracings/volume/c533a7b1-63fe-4ea2-92d6-249c8d40d0e4/duplicate?token=<redacted>] ->
Dec 06 18:42:24 data1-brain docker-compose[21770]: webknossos-tracingstore-mhlab_1  |
Dec 06 18:42:24 data1-brain docker-compose[21770]: webknossos-tracingstore-mhlab_1  | play.api.UnexpectedException: Unexpected exception[NoSuchElementException: None.get]
Dec 06 18:42:24 data1-brain docker-compose[21770]: webknossos-tracingstore-mhlab_1  |         at play.api.http.HttpErrorHandlerExceptions$.throwableToUsefulException(HttpErrorHandler.scala:247)
Dec 06 18:42:24 data1-brain docker-compose[21770]: webknossos-tracingstore-mhlab_1  |         at play.api.http.DefaultHttpErrorHandler.onServerError(HttpErrorHandler.scala:178)
Dec 06 18:42:24 data1-brain docker-compose[21770]: webknossos-tracingstore-mhlab_1  |         at play.core.server.AkkaHttpServer$$anonfun$1.applyOrElse(AkkaHttpServer.scala:363)
Dec 06 18:42:24 data1-brain docker-compose[21770]: webknossos-tracingstore-mhlab_1  |         at play.core.server.AkkaHttpServer$$anonfun$1.applyOrElse(AkkaHttpServer.scala:361)
Dec 06 18:42:24 data1-brain docker-compose[21770]: webknossos-tracingstore-mhlab_1  |         at scala.concurrent.Future.$anonfun$recoverWith$1(Future.scala:413)
Dec 06 18:42:24 data1-brain docker-compose[21770]: webknossos-tracingstore-mhlab_1  |         at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:37)
Dec 06 18:42:24 data1-brain docker-compose[21770]: webknossos-tracingstore-mhlab_1  |         at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:60)
Dec 06 18:42:24 data1-brain docker-compose[21770]: webknossos-tracingstore-mhlab_1  |         at akka.dispatch.BatchingExecutor$AbstractBatch.processBatch(BatchingExecutor.scala:55)
Dec 06 18:42:24 data1-brain docker-compose[21770]: webknossos-tracingstore-mhlab_1  |         at akka.dispatch.BatchingExecutor$BlockableBatch.$anonfun$run$1(BatchingExecutor.scala:91)
Dec 06 18:42:24 data1-brain docker-compose[21770]: webknossos-tracingstore-mhlab_1  |         at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
Dec 06 18:42:24 data1-brain docker-compose[21770]: webknossos-tracingstore-mhlab_1  |         at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:81)
Dec 06 18:42:24 data1-brain docker-compose[21770]: webknossos-tracingstore-mhlab_1  |         at akka.dispatch.BatchingExecutor$BlockableBatch.run(BatchingExecutor.scala:91)
Dec 06 18:42:24 data1-brain docker-compose[21770]: webknossos-tracingstore-mhlab_1  |         at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:40)
Dec 06 18:42:24 data1-brain docker-compose[21770]: webknossos-tracingstore-mhlab_1  |         at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(ForkJoinExecutorConfigurator.scala:43)
Dec 06 18:42:24 data1-brain docker-compose[21770]: webknossos-tracingstore-mhlab_1  |         at akka.dispatch.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
Dec 06 18:42:24 data1-brain docker-compose[21770]: webknossos-tracingstore-mhlab_1  | Caused by: java.util.NoSuchElementException: None.get
Dec 06 18:42:24 data1-brain docker-compose[21770]: webknossos-tracingstore-mhlab_1  |         at scala.None$.get(Option.scala:349)
Dec 06 18:42:24 data1-brain docker-compose[21770]: webknossos-tracingstore-mhlab_1  |         at scala.None$.get(Option.scala:347)
Dec 06 18:42:24 data1-brain docker-compose[21770]: webknossos-tracingstore-mhlab_1  |         at com.scalableminds.webknossos.tracingstore.tracings.volume.VersionedBucketIterator.next(VolumeTracingBucketHelper.scala:76)
Dec 06 18:42:24 data1-brain docker-compose[21770]: webknossos-tracingstore-mhlab_1  |         at com.scalableminds.webknossos.tracingstore.tracings.volume.BucketIterator.next(VolumeTracingBucketHelper.scala:103)
Dec 06 18:42:24 data1-brain docker-compose[21770]: webknossos-tracingstore-mhlab_1  |         at com.scalableminds.webknossos.tracingstore.tracings.volume.BucketIterator.next(VolumeTracingBucketHelper.scala:99)
Dec 06 18:42:24 data1-brain docker-compose[21770]: webknossos-tracingstore-mhlab_1  |         at scala.collection.Iterator$$anon$10.next(Iterator.scala:455)
Dec 06 18:42:24 data1-brain docker-compose[21770]: webknossos-tracingstore-mhlab_1  |         at scala.collection.Iterator.foreach(Iterator.scala:937)
Dec 06 18:42:24 data1-brain docker-compose[21770]: webknossos-tracingstore-mhlab_1  |         at scala.collection.Iterator.foreach$(Iterator.scala:937)
Dec 06 18:42:24 data1-brain docker-compose[21770]: webknossos-tracingstore-mhlab_1  |         at scala.collection.AbstractIterator.foreach(Iterator.scala:1425)
Dec 06 18:42:24 data1-brain docker-compose[21770]: webknossos-tracingstore-mhlab_1  |         at scala.collection.generic.Growable.$plus$plus$eq(Growable.scala:58)
Dec 06 18:42:24 data1-brain docker-compose[21770]: webknossos-tracingstore-mhlab_1  |         at scala.collection.generic.Growable.$plus$plus$eq$(Growable.scala:49)
Dec 06 18:42:24 data1-brain docker-compose[21770]: webknossos-tracingstore-mhlab_1  |         at scala.collection.mutable.ListBuffer.$plus$plus$eq(ListBuffer.scala:185)
Dec 06 18:42:24 data1-brain docker-compose[21770]: webknossos-tracingstore-mhlab_1  |         at scala.collection.mutable.ListBuffer.$plus$plus$eq(ListBuffer.scala:43)
Dec 06 18:42:24 data1-brain docker-compose[21770]: webknossos-tracingstore-mhlab_1  |         at scala.collection.TraversableOnce.to(TraversableOnce.scala:309)
Dec 06 18:42:24 data1-brain docker-compose[21770]: webknossos-tracingstore-mhlab_1  |         at scala.collection.TraversableOnce.to$(TraversableOnce.scala:307)
```

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -166,7 +166,7 @@ jobs:
       - run:
           name: Run webknossos-tracingstore smoke test
           command: |
-            sleep 10
+            sleep 30
             curl --fail --retry 3 --max-time 15 -v http://localhost:9050/tracings/health
       - run:
           name: Stop webknossos-tracingstore

--- a/webknossos-tracingstore/app/com/scalableminds/webknossos/tracingstore/tracings/volume/VolumeTracingBucketHelper.scala
+++ b/webknossos-tracingstore/app/com/scalableminds/webknossos/tracingstore/tracings/volume/VolumeTracingBucketHelper.scala
@@ -77,12 +77,14 @@ class VersionedBucketIterator(prefix: String, volumeDataStore: FossilDBClient, v
   }
 
   private def parseBucketKey(key: String): Option[(String, BucketPosition)] = {
-    val keyRx = "([0-9a-z-]+)/(\\d+)/(\\d+)-\\[\\d+,\\d+,\\d+\\]".r
+    val keyRx = "([0-9a-z-]+)/(\\d+)/-?\\d+-\\[(\\d+),(\\d+),(\\d+)\\]".r
 
     key match {
-      case keyRx(name, res, morton) =>
-        val resolution = res.toInt
-        val (x, y, z) = mortonDecode(morton.toLong)
+      case keyRx(name, resolutionStr, xStr, yStr, zStr) =>
+        val resolution = resolutionStr.toInt
+        val x = xStr.toInt
+        val y = yStr.toInt
+        val z = zStr.toInt
         val bucket = new BucketPosition(
           x * resolution * DataLayer.bucketLength,
           y * resolution * DataLayer.bucketLength,

